lib/index.js

```diff
@@ -60,4 +97,7 @@ function resetSearchField() {
 function clearSearchResults() {
   document.querySelector(DOMstrings.searchResults).innerHTML = "";
   document.querySelector(DOMstrings.searchedArtist).innerHTML = "";
+  if (trackObjArray !== undefined) {
+    trackObjArray = []
```

Comment:
Why is this check necessary?

Reply:
Per discussion - this clears the global `trackObjArray` variable between searches